File: atinject-integration/src/main/resources/webapp/scripts/atinject/user/GetUserResponse.js
define(["atinject/WebSocketResponse", "atinject/user/User"],
function(WebSocketResponse, User){
    function GetUserResponse() {
        this._class = "org.atinject.api.user.dto.GetUserResponse"
    };

    GetUserResponse.prototype = new WebSocketResponse();

        GetUserResponse.prototype.getUser = function() {
        return this.user;
    };

    GetUserResponse.prototype.setUser = function(user) {
        this.user = user;
        return this;
    };


    
    return GetUserResponse;
});

File: atinject-api/atinject-api-user/src/main/java/org/atinject/api/user/dto/GetUserResponse.java
package org.atinject.api.user.dto;

import org.atinject.core.websocket.dto.WebSocketResponse;

public class GetUserResponse extends WebSocketResponse
{
    private static final long serialVersionUID = 1L;

    private User user;

    public User getUser()
    {
        return user;
    }

    public GetUserResponse setUser(User user)
    {
        this.user = user;
        return this;
    }
    
}
